packages/block-library/src/navigation-link/edit.js

```diff
@@ -320,7 +322,7 @@ export default function NavigationLinkEdit( {
 			} ),
 		},
 		{
-			allowedBlocks: [ 'core/navigation-link', 'core/spacer' ],
```

Comment:
I wonder what we can do to communicate to block creators that this should be a constant or at least memoized.